src/apollo/zimbra-in-memory-cache.ts

```diff
@@ -15,6 +15,8 @@ const dataIdFromPath = (result: any, path: string) => {
 
 const dataIdFromObject = (object: any): string | null | undefined => {
 	switch (object.__typename) {
+		case 'CalendarItemHitInfo':
```

Comment:
Great fix !!!
Thanks @lavegupta @itsmeprashant :100:

Reply:
All credit goes to @itsmeprashant